site/static/compare/script.js

```diff
@@ -363,8 +373,16 @@ app.component('test-cases-table', {
         },
     },
     template: `
-<div class="bench-table">
-<div class="category-title">{{ title }} benchmarks</div>
+<div class="bench-table" :id="id">
+<div class="category-title">
+  {{ title }} benchmarks
+  <span v-if="title == 'Primary'" title="To secondary benchmarks">
+    <a href="#secondary-benchmarks">⮯</a>
+  </span>
+  <span v-else title="To primary benchmarks">
+    <a href="#primary-benchmarks">⮭</a>
+  </span>
```

Comment:
I don't love that we have somewhat complex logic based on the title. It makes this component harder to understand. Can we do something like this instead?

```suggestion
  <span :title="To {title} benchmarks">
    <a :href="#{title}-benchmarks">⮯</a>
  </span>
```

Reply:
Ah, but then I would need to add yet another prop to say if button should point up or down :)